tests/run_bf.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+# SPDX-FileCopyrightText: 2020 the prplMesh contributors (see AUTHORS.md)
+# This code is subject to the terms of the BSD+Patent license.
+# See LICENSE file for more details.
+
+scriptdir=$(dirname "$(readlink -f "${0}")")
+bf_plugins_dir=${scriptdir}/boardfarm_plugins
+bf_dir=$(realpath "${scriptdir}/../../boardfarm/boardfarm")
+
+if [ -n "${PYTHONPATH}" ]; then
+   PYTHONPATH="${bf_dir}:${bf_plugins_dir}:${scriptdir}:${PYTHONPATH}"
+else
+   PYTHONPATH="${bf_dir}:${bf_plugins_dir}:${scriptdir}"
+fi
+export PYTHONPATH
+
+exec bft -c "${bf_plugins_dir}"/boardfarm_prplmesh/prplmesh_config.json -n prplmesh_docker -x test_flows
```

Comment:
```suggestion
exec ${bf_dir}/bft -c "${bf_plugins_dir}"/boardfarm_prplmesh/prplmesh_config.json -n prplmesh_docker -x test_flows
```

With those two changes, it works for me without calling `setup.py`.